extensions/wikia/Email/Controller/FacebookDisconnectController.class.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Email\Controller;
+
+use Email\EmailController;
+use Email\Tracking\TrackingCategories;
+
+class FacebookDisconnectController extends ForgotPasswordController {
+	const TRACKING_CATEGORY = TrackingCategories::TEMPORARY_PASSWORD;
```

Comment:
Don't need to define `TRACKING_CATEGORY` here since it's inherited from `ForgotPasswordController`.